helm/hpcc/values.schema.json

```diff
@@ -426,7 +426,19 @@
         "defaultSprayParts" : {
             "description": "Number of parts sprayed by default",
             "type" : "integer"
+        },
+        "cost" : {
+            "description": "Costs associated with the storage and use of the plane",
+            "type" : "object",
+            "default": "0.1193"
```

Comment:
missing comma, I think this will fail cause helm install to fail.

Also, this is default for this "object" type, I think this line is supposed to be under "storageAtRest" ?

Reply:
sorry, that was careless.

Reply:
I have removed the "default" attribute because it was being ignored.